README.md

```diff
@@ -55,17 +61,365 @@ npm i @xmtp/react-native-sdk
 npx pod-install
 ```
 
+<!--any thoughts on providing more details here?-->
+
 We're working on testing the end-to-end installation and will provide more platform-specific configuration details.
 
 ### Configure for Android
 
 Your app must use Android `minSdkVersion = 22` to work with the `xmtp-react-native` SDK.
 
+<!--any thoughts on providing more details here?-->
+
 We're working on testing the end-to-end installation and will provide more platform-specific configuration details.
 
-## Enable the example app to send push notifications
+## Usage
+
+The [XMTP message API](https://xmtp.org/docs/concepts/architectural-overview#network-layer) revolves around a network client that allows retrieving and sending messages to other network participants. A client must be connected to a wallet on startup. If this is the very first time the client is created, the client will generate a [key bundle](https://xmtp.org/docs/concepts/key-generation-and-usage) that is used to [encrypt and authenticate messages](https://xmtp.org/docs/concepts/invitation-and-message-encryption). The key bundle persists encrypted in the network using a [wallet signature](https://xmtp.org/docs/concepts/account-signatures). The public side of the key bundle is also regularly advertised on the network to allow parties to establish shared encryption keys. All this happens transparently, without requiring any additional code.
+
+<!--as discussed, use the README from xmtp-js as the basis for this README. In the code samples, I changed @xmtp/xmtp-js in the imports to @xmtp/xmtp-react-native - correct?-->
+
+<!--I also changed the language in the code fences from tsx to js - correct?-->
+
+```js
+import { Client } from '@xmtp/xmtp-react-native'
+import { Wallet } from 'ethers'
+
+// You'll want to replace this with a wallet from your application
+const wallet = Wallet.createRandom()
+// Create the client with your wallet. This will connect to the XMTP development network by default
+const xmtp = await Client.create(wallet)
+// Start a conversation with XMTP
+const conversation = await xmtp.conversations.newConversation(
+  '0x3F11b27F323b62B159D2642964fa27C46C841897'
+)
+// Load all messages in the conversation
+const messages = await conversation.messages()
+// Send a message
+await conversation.send('gm')
+// Listen for new messages in the conversation
+for await (const message of await conversation.streamMessages()) {
+  console.log(`[${message.senderAddress}]: ${message.content}`)
+}
+```
+
+Currently, network nodes are configured to rate limit high-volume publishing from clients. A rate-limited client can expect to receive a 429 status code response from a node. Rate limits can change at any time in the interest of maintaining network health.
+
+## Create a client
+
+<!--Signer.ts doesn't exist in this repo. How might we update this text below?-->
```

Comment:
See comment

Reply:
We use third web in our example but they could use whatever

```
import { ConnectWallet, useSigner } from "@thirdweb-dev/react-native";

const client = await XMTP.Client.create(useSigner(), "dev");

```